src/pages/PeoplePage.tsx

```diff
@@ -0,0 +1,67 @@
+import React, { useState, useEffect } from 'react';
+import { getPeople } from '../api';
+import { Person } from '../types';
+import { Loader } from '../components/Loader';
+import { PeopleTable } from '../components/PeopleTable';
+
+const peopleWithParents = (people: Person[]): Person[] => {
+  return people.map(person => {
+    const mother = people.find(({ name }) => name === person.motherName);
+    const father = people.find(({ name }) => name === person.fatherName);
+
+    return ({
+      ...person,
+      mother,
+      father,
+    });
+  });
+};
+
+export const PeoplePage: React.FC = () => {
+  const [people, setPeople] = useState<Person[]>([]);
+  const [loading, setLoading] = useState(false);
+  const [error, setError] = useState(false);
+
+  useEffect(() => {
+    setLoading(true);
```

Comment:
set `loading` `true` as initial value to avoid micro jumping of your content
```suggestion
  const [loading, setLoading] = useState(true);
  const [error, setError] = useState(false);

  useEffect(() => {
```